Python/tier2_redundancy_eliminator_bytecodes.c

```diff
@@ -84,9 +84,7 @@ dummy_func(void) {
             assert(PyLong_CheckExact(get_const(right)));
             PyObject *temp = _PyLong_Add((PyLongObject *)get_const(left),
                                          (PyLongObject *)get_const(right));
```

Comment:
Maybe there should be s `get_long_const()` helper which returns `(PyLongObject *)get_const(x)` and also includes the `assert(PyLong_CheckExact())` call, to make code like this more compact. (Same for float.)